packages/config-schema/tests/fixtures/edge-functions-pass/test.toml

```diff
@@ -0,0 +1,7 @@
+[[edge_functions]]
+path = "/admin"
+function = "auth"
```

Comment:
https://github.com/netlify/build/blob/main/packages/config/src/edge_functions.ts#L45

`cache` and potential more fields missing, like `excluded_path`